typings/index.d.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference types="node" />
+export { link, LinkError, Receiver, Sender } from "./link";
+export {
+  AmqpError, AmqpMessage, AmqpMessageAnnotations, AmqpMessageProperties,
+  Connection, ConnectionOptions, EventContext, DeliveryAnnotations, Dictionary,
+  EndpointOptions, LinkOptions, ReceiverOptions, SenderOptions, TerminusOptions
+} from "./connection";
+export { ConnectionError, ProtocolError, TypeError } from "./errors";
+export { Delivery, Session } from "./session";
+export { Message, Section, message as IMessage } from "./message";
```

Comment:
Message and Section should not be exported. Having IMessage and AmqpMessage is confusing. I think AmqpMessage should be renamed Message and IMessage renamed something like MessageUtil.

Reply:
done. I would leave the AmqpError as-is, otherwise it will collide with the Javascript Error object.